GiniSwitchSDK/Example/Tests/ExtractionResourcesTests.swift

```diff
@@ -62,7 +62,7 @@ class ExtractionResourcesTests: XCTestCase {
         let testOrder = "https://switch.gini.net/extractionOrders/\(testOrderId)/pages"
         let extractionsResource = service.statusFor(orderUrl:testOrder)
         XCTAssertEqual(extractionsResource.url, URL(string: testOrder), "The extractions status request URL doesn't match")
-        XCTAssertEqual(extractionsResource.method, "GET", "The extractions status request method doesn't match")
+        XCTAssertEqual(extractionsResource.method, .GET, "The extractions status request method doesn't match")
```

Comment:
Enum cases in Swift should be lowercased, but I'm ok with this.

Reply:
I thought about it, but on the other hand people are used to seeing the HTTP methods in capital letters. I also don't have a strong opinion though.